thetis/options.py

```diff
@@ -124,8 +124,34 @@ class SemiImplicitTimestepperOptions3d(ExplicitTimestepperOptions3d):
         help='implicitness parameter theta for 2D solver. Value 1.0 implies fully implicit formulation.').tag(config=True)
 
 
-class GLSModelOptions(FrozenHasTraits):
-    """Options for generic length scale turbulence model"""
+class TurbulenceModelOptions(FrozenHasTraits):
```

Comment:
Could we perhaps define this in a more general Options class from which all other Options classes are derived (instead of FrozenHasTraits), ModelOptions2D, TimestepperOptions, etc. - there is nothing turbulence-model-specific about that update() function and could be useful elsewhere.